internal/providers/terraform/azure/testdata/hdinsight_kafka_cluster_test/hdinsight_kafka_cluster_test.golden

```diff
@@ -0,0 +1,33 @@
+
+ Name                                            Monthly Qty  Unit                  Monthly Cost 
+                                                                                                 
+ azurerm_hdinsight_kafka_cluster.non_usage                                                       
+ ├─ Head node (NC24)                                   1,460  hours                    $7,478.12 
+ ├─ Worker node (D3)                                     730  hours                      $215.35 
+ ├─ Zookeeper node (D3)                                2,190  hours                      $646.05 
+ ├─ Managed OS disks                                       3  months                     $446.04 
+ └─ Disk operations                         Cost depends on usage: $0.00036 per 100K operations  
```

Comment:
Need to rebase and regenerate so you pick up the "Monthly cost depends..." change.